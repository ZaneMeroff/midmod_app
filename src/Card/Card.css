#card {
  background-color: #FFF;
  margin: 1%;
  border-radius: 10px;
  width: 250px;
  border: 2px solid #000;
}
