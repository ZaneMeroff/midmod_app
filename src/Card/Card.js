import React from 'react';
import './Card.css'

const Card = props => {

  return (
    <div id='card'>
      <h4>Name: {props.name}</h4>
      <h4>Date: {props.date}</h4>
      <h4>Time: {props.time}</h4>
      <h4>Size of Party: {props.number}</h4>
      <h5>id: {props.id}</h5>
      <button>cancel</button>
    </div>
  )

}

export default Card;
