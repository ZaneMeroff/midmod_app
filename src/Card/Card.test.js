import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import { shallow } from 'enzyme';

describe('Card', () => {
  let wrapper;
  let mockRes = jest.fn();

  beforeEach(() => {
    wrapper = shallow(<Card
      name='mockName'
      date='mockDate'
      time='mockTime'
      number='mockNumber'
      id='mockId'
    />)
  })

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})
