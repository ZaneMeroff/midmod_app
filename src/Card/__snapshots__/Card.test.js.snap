// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot 1`] = `
<div
  id="card"
>
  <h4>
    Name: 
    mockName
  </h4>
  <h4>
    Date: 
    mockDate
  </h4>
  <h4>
    Time: 
    mockTime
  </h4>
  <h4>
    Size of Party: 
    mockNumber
  </h4>
  <h5>
    id: 
    mockId
  </h5>
  <button>
    cancel
  </button>
</div>
`;
